lib/Installation/Popups/YesNoPopup.pm

```diff
@@ -22,7 +22,7 @@ sub new {
 sub init {
     my $self = shift;
     $self->{btn_yes} = $self->{app}->button({id => 'yes'});
-    $self->{btn_no} = $self->{app}->button({id => 'no'});
+    $self->{btn_no} = $self->{app}->button({id => qr/^no$|^no_button$/});
```

Comment:
if you still like this idea, this is more clear to read: `^(no|no_button)$

Reply:
Changed that and restarted all VRs.